src/ui/units/workbooks/components/Table/WorkbookEntriesTable/MainTabContent/MainTabContent.tsx

```diff
@@ -0,0 +1,137 @@
+import {Button, Icon} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+import {I18n} from 'i18n';
+import {useDispatch} from 'react-redux';
+import {CreateEntryActionType} from 'ui/units/workbooks/constants';
+import {setCreateWorkbookEntryType} from 'ui/units/workbooks/store/actions';
+
+import {ChunkGroup} from '../ChunkGroup/ChunkGroup';
+import {WorkbookEntriesTableProps} from '../types';
+import {ChunkItem} from '../useChunkedEntries';
+
+import './MainTabContent.scss';
+
+const b = block('dl-main-tab-content');
+
+const i18n = I18n.keyset('new-workbooks');
+
+interface MainTabContentProps extends WorkbookEntriesTableProps {
+    chunk: ChunkItem[];
+    actionCreateText: string;
+    title: string;
+    isErrorMessage?: boolean;
+    actionType: CreateEntryActionType;
+    isShowMoreBtn: boolean;
+    loadMoreEntries: () => void;
+    retryLoadEntries: () => void;
+}
+
+const MainTabContent: React.FC<MainTabContentProps> = ({
+    workbook,
+    chunk,
+    onRenameEntry,
+    onDeleteEntry,
+    onDuplicateEntry,
+    onCopyEntry,
+    actionCreateText,
+    title,
+    actionType,
+    isShowMoreBtn,
+    loadMoreEntries,
+    retryLoadEntries,
+    isErrorMessage,
+}) => {
+    const [isOpen, setIsOpen] = React.useState(true);
+
+    const dispatch = useDispatch();
+
+    const handleCreateEntity = () => {
+        dispatch(setCreateWorkbookEntryType(actionType));
+    };
+
+    const getContentTab = () => {
+        if (isErrorMessage)
+            return <div className={b('error-text')}>{i18n('label_error-load-entities')}</div>;
+
+        if (chunk.length > 0 && isOpen) {
+            return (
+                <ChunkGroup
+                    key={chunk[0].key}
+                    workbook={workbook}
+                    chunk={chunk}
+                    onRenameEntry={onRenameEntry}
+                    onDeleteEntry={onDeleteEntry}
+                    onDuplicateEntry={onDuplicateEntry}
+                    onCopyEntry={onCopyEntry}
+                />
+            );
+        }
+
+        if (isOpen) return <div className={b('no-objects')}>{i18n('no_objects')}</div>;
+
+        return null;
+    };
+
+    const getActionBtn = () => {
+        if (isErrorMessage) {
+            return (
+                <Button onClick={retryLoadEntries} className={b('retry-btn')} view="outlined">
+                    {i18n('action_retry')}
+                </Button>
+            );
+        }
+
+        if (isShowMoreBtn && isOpen) {
+            return (
+                <Button onClick={loadMoreEntries} className={b('show-more-btn')} view="outlined">
+                    {i18n('action_show-more')}
+                </Button>
+            );
+        }
+
+        return null;
+    };
+
+    return (
+        <div className={b()}>
+            <div className={b('header', {closed: !isOpen})}>
+                <div className={b('content')}>
+                    <div className={b('title')}>
```

Comment:
Can we put `onClick={() => setIsOpen(!isOpen)}` here for making entries title clickable too?
And add `cursor: pointer`.

<img width="298" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/f260fa09-5024-4ab2-b80a-eabd3cab2e47">

Reply:
yes, added onClick for title, and removed class for btn